src/extensions/displayExtensionInfo.ts

```diff
@@ -54,33 +103,29 @@ export async function displayExtInfo(
   if (roles.length) {
     lines.push(await displayRoles(roles));
   }
-  if (lines.length > 0) {
-    utils.logLabeledBullet(logPrefix, `information about '${clc.bold(extensionName)}':`);
-    const infoStr = lines.join("\n");
-    // Convert to markdown and convert any trailing newlines to a single newline.
-    const formatted = marked(infoStr).replace(/\n+$/, "\n");
-    logger.info(formatted);
-    // Return for testing purposes.
-    return lines;
-  } else {
-    throw new FirebaseError(
-      "Error occurred during installation: cannot parse info from source spec",
-      {
-        context: {
-          spec: spec,
-          extensionName: extensionName,
-        },
-      }
-    );
-  }
+  logger.info(`\n${lines.join("\n")}`);
+  return lines;
 }
 
-/**
- * Prints a clickable link where users can download the source code for an Extension Version.
- */
-export function printSourceDownloadLink(sourceDownloadUri: string): void {
-  const sourceDownloadMsg = `Want to review the source code that will be installed? Download it here: ${sourceDownloadUri}`;
-  utils.logBullet(marked(sourceDownloadMsg));
+export function displayEvents(spec: ExtensionSpec) {
+  const lines = spec.events?.map((event: EventDescriptor) => {
+    return `  - ${clc.magenta(event.type)}${event.description ? `: ${event.description}` : ""}`;
+  });
+  return (
+    clc.bold("Events emitted by this extension:\n") + (lines?.length ? lines.join("\n") : " - None")
+  );
+}
+
+export function displayResources(spec: ExtensionSpec) {
```

Comment:
This doesn't take TaskQueue resources into account. They are automatically created for "taskQueueTrigger" functions.
This also doesn't take Secrets into account. They are under params with type "secret".
We show both of these types of resources when installing through the console. They should be here too.

Reply:
Good eye! I'll ensure parity with console.